app/main.py

```diff
@@ -2,0 +1,46 @@
+class Animal:
+    alive = []
```

Comment:
Consider adding a type annotation for the `alive` list, such as `alive: list[Animal] = []`, to improve code clarity and maintainability.